services/brig/src/Brig/User/API/Search.hs

```diff
@@ -49,16 +54,49 @@ routesPublic = do
       .&. header "Z-User"
       .&. query "q"
       .&. def (unsafeRange 15) (query "size")
+
   document "GET" "search" $ do
     Doc.summary "Search for users"
     Doc.parameter Doc.Query "q" Doc.string' $
       Doc.description "Search query"
     Doc.parameter Doc.Query "size" Doc.int32' $ do
       Doc.description "Number of results to return (min: 1, max: 500, default: 15)"
       Doc.optional
-    Doc.returns (Doc.ref Public.modelSearchResult)
+    Doc.returns (Doc.ref $ Public.modelSearchResult Public.modelSearchContact)
     Doc.response 200 "The search result." Doc.end
 
+  get "/teams/:tid/search" (continue browseTeamH) $
```

Comment:
With this route, should it be moved to the module where the other routes under `/teams/:tid/` are stored?

Donno, our routing table tree is a bit out of sorts anyway, so I'm not sure how to keep this from getting worse.  Leaving it here makes sense because it's also about searching.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3359/files#diff-011d7779c23ee1ccc3e3cfbda90c1e55091483907d4732773fd5c53bc5236ed5R49

Generally `/teams/{tid}/...` makes sense when you need the `teamId`. That would match the pattern for most of the team endpoint looking from the outside.

Reply:
I'd also prefer keeping it closer to search and the cost finding mapping of path -> module is quite low